### ISSUE TITLE
[ObjectStatus]: A11y-ObjectStatus fails to meet 4:5:1 contrast ratio

### ISSUE THREAD

**Issue by I585157** [SAP/ui5-webcomponents-react#5079]
### Describe the bug

[ObjectStatus]: Inverted Object status with ValueState.Error has a contrast ratio of 3:9:1, failing to meet the expected 4:5:1 contrast ratio. 

### Isolated Example

https://sap.github.io/ui5-webcomponents-react/?path=/story/data-display-objectstatus--inverted

### Reproduction steps

1. examine the foreground color for the ValueState.Error ObjectStatus object
2. examine the background color for the ValueState.Error ObjectStatus object
3. compute the color contrast ratio
...

### Expected Behaviour

Expected ratio should be at least 4:5:1.

### Screenshots or Videos

<img width="957" alt="Screenshot 2023-09-20 at 2 20 15 PM" src="https://github.com/SAP/ui5-webcomponents-react/assets/127151629/ee5ce469-85fc-4b4b-8632-9ad8abd635cf">

### UI5 Web Components for React Version

1.17.0

### UI5 Web Components Version

ObjectStatus

### Browser

Chrome

### Operating System

Mac OS

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
The design specs of the `ObjectStatus` where updated, unfortunately we now have to wait before we can implement them for the `theming-base-content` version to be updated.

**Comment by Lukas742** [comment]
Hi @I585157 

after applying the new variables I noticed that the colors for the inverted `ValueState.Error` didn't change, so this seems to be implemented by design. Please also note that a text-shadow is used here to increase contrast-ratio and readability. If you want to challenge that, please reach out to our design and a11y experts. You can contact me internally if you need contact information. 

Since we are using the official CSS variables, I will close this issue now. If there are any further developments on this topic, please feel free to comment here.
